doc/check-options.md

```diff
@@ -0,0 +1,215 @@
+| ------------ | :--------------------- | :---------------------------------------- |
+| `attributes` | `['lang', 'xml:lang']` | Attributes to check for valid lang values |
+
+### frame-tested
+
+| Option        | Default | Description                                                |
+| ------------- | :------ | :--------------------------------------------------------- |
+| `isViolation` | `false` | If an iframe without axe should be reported as a violation |
+
+### no-autoplay-audio
+
+| Option            | Default | Description                                                                   |
+| ----------------- | :------ | :---------------------------------------------------------------------------- |
+| `allowedDuration` | `3`     | Maximum time an audio is allowed to autoplay before being marked as violation |
+
+### css-orientation-lock
+
+| Option            | Default | Description                                                                                                                            |
+| ----------------- | :------ | :------------------------------------------------------------------------------------------------------------------------------------- |
+| `degreeThreshold` | `3`     | The difference of degrees from 180 and 90 that are considered orientation lock (for example, 93° is orientation locked but 94° is not) |
+
+### meta-viewport-large
+
+| Option         | Default | Description                                                                                                |
+| -------------- | :------ | :--------------------------------------------------------------------------------------------------------- |
+| `scaleMinimum` | `5`     | The maximum CSS value of `scale-maximum` the check will apply to. Values above this number will be ignored |
+| `lowerBound`   | `2`     | The minimum CSS value of `scale-minimum` the check will apply to. Values below this number will be ignored |
+
+### meta-viewport
+
+| Option         | Default | Description                                                                                                |
+| -------------- | :------ | :--------------------------------------------------------------------------------------------------------- |
+| `scaleMinimum` | `2`     | The maximum CSS value of `scale-maximum` the check will apply to. Values above this number will be ignored |
+
+### header-present
+
+| Option     | Default                                                                                                          | Description                                        |
+| ---------- | :--------------------------------------------------------------------------------------------------------------- | :------------------------------------------------- |
+| `selector` | `h1:not([role]), h2:not([role]), h3:not([role]), h4:not([role]), h5:not([role]), h6:not([role]), [role=heading]` | Selector used to determine if a page has a heading |
```

Comment:
Hard to read. Same problem as big line of inline CSS above; it's hard to read. Anything we can do to make it clear that this is CSS to make it easier to read is appreciated. (I know this is a CSS selector in theory, but the inline presentation munges stuff together and makes it less obvious that we're talking about a series of very specific selectors here.